src/mono_camera.cpp

```diff
@@ -67,9 +66,13 @@ MonoCamera::MonoCamera(ros::NodeHandle& nh, ros::NodeHandle& nhp) : nh_(nh), nhp
 MonoCamera::~MonoCamera(void) {
   cam_.stop();
   pub_.shutdown();
-}
+  cam_status=cav_msgs::DriverStatus::OFF;
```

Comment:
If the destructor is called, the ros connect should already be closed. No need to change status here.